lib/stripe.js

```diff
@@ -133,6 +135,26 @@ Stripe.prototype = {
     );
   },
 
+  setAppInfo: function(info) {
+    if (info && typeof info !== 'object') {
+      throw new Error('AppInfo must be an object.');
+    }
+
+    // Kill the cached UA string because it may no longer be valid
+    Stripe.USER_AGENT_SERIALIZED = undefined;
+
+    this._appInfo = APP_INFO_PROPERTIES.reduce(function(accum, prop) {
```

Comment:
I like the format checking here. Is there something that we can do to make `name` required? I believe that's what we're doing in most of the other bindings.